From 4cde8bae49275edb2815b98cc3404238bb5799dd Mon Sep 17 00:00:00 2001
From: Diego Biurrun <diego@biurrun.de>
Date: Wed, 27 Aug 2014 13:14:20 +0200
Subject: [PATCH 31/37] license: Mention that vf_interlace is GPL, not LGPL

(cherry picked from commit 9e8bbe7d4d1dcd5fec491dbfbb98ed2038a7bed5)
Signed-off-by: Diego Biurrun <diego@biurrun.de>
---
 LICENSE | 1 +
 1 file changed, 1 insertion(+)

diff --git a/LICENSE b/LICENSE
index fb29178..701e6e1 100644
--- a/LICENSE
+++ b/LICENSE
@@ -21,6 +21,7 @@ Specifically, the GPL parts of Libav are
     - vf_cropdetect.c
     - vf_delogo.c
     - vf_hqdn3d.c
+    - vf_interlace.c
 
 Should you, for whatever reason, prefer to use version 3 of the (L)GPL, then
 the configure parameter --enable-version3 will activate this licensing option
-- 
1.9.1

